napari/layers/image/image.py

```diff
@@ -792,6 +792,30 @@ def on_chunk_loaded(self, request: ChunkRequest) -> None:
             data = self._SliceDataClass.from_request(self, request)
             self._on_data_loaded(data, sync=False)
 
+    def _get_value_3d(
+        self,
+        start_position: np.ndarray,
+        end_position: np.ndarray,
+        dims_displayed: List[int],
+    ):
+        """Get the layer data value along a ray
+
+        Parameters
+        ----------
+        start_position : np.ndarray
+            The start position of the ray used to interrogate the data.
+        end_position : np.ndarray
+            The end position of the ray used to interrogate the data.
+        dims_displayed : List[int]
+            The indices of the dimensions currently displayed in the Viewer.
+
+        Returns
+        -------
+        value
+            The data value along the supplied ray.
+        """
+        return None
```

Comment:
Perhaps grab the rendering mode and compute the value that way? At least for min, mean, max, and attenuated max projection?

Reply:
I definitely think we should get the value in a way that matches the shader. I think adding it for the `Image` layer will be a separate PR though.